### PR TITLE
Be able to configure job container image

### PR DESCRIPTION
**Pull request by kkaneda**
This is useful for local testing.

### CODE REVIEW

File: deployments/dispatcher/values.yaml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 jobPollingInterval: 10s
 jobNamespace: default
-jobVersion:
+job:
+  image: public.ecr.aws/v8n3t7y5/llm-operator/fine-tuning
+  fakeImage: public.ecr.aws/v8n3t7y5/llm-operator/fake-job
+  version: latest
```
Comment (jmuk) [inline deployments/dispatcher/values.yaml:6]: we do not have latest tag (yet) so please leave this as empty.

File: dispatcher/internal/config/config.go [hunk 2]
```diff
@@ -49,11 +49,33 @@ type KubernetesManagerConfig struct {
 	PprofBindAddress   string `yaml:"pprofBindAddress"`
 }
 
+// JobConfig is the job configuration.
+type JobConfig struct {
+	Image     string `yaml:"image"`
+	FakeImage string `yaml:"fakeImage"`
```
Comment (jmuk) [inline dispatcher/internal/config/config.go:55]: It is not clear to me why we should keep both fake image and normal image -- I guess it's okay to have a single image, and for debugging simply set the fake image URL to the image config.

File: deployments/dispatcher/values.yaml (hunk 1 above)
Comment (kkaneda) [reply deployments/dispatcher/values.yaml:6]: done

File: dispatcher/internal/config/config.go (hunk 2 above)
Comment (kkaneda) [reply dispatcher/internal/config/config.go:55]: sorry, could you elaborate?

Comment (kkaneda) [reply dispatcher/internal/config/config.go:55]: Ah i now see what you meant. It was just for convenient, but let me change.

Comment (kkaneda) [reply dispatcher/internal/config/config.go]: done

**Review by jmuk — APPROVED** [review]

**Pull request merged**